tests/test_latentpositiontest.py

```diff
@@ -1,79 +1,79 @@
-# Copyright (c) Microsoft Corporation and contributors.
-# Licensed under the MIT License.
-
 import unittest
+import pytest
 import numpy as np
-from graspologic.inference import LatentPositionTest
+
+from graspologic.inference import latent_position_test, _difference_norm
```

Comment:
not what i meant. it's still importable from inference. i meant to import from the file itself, without adding it to the init at all.